### PR TITLE
Accelerometer

### PR DESCRIPTION
**Pull request by RikardO-HM**

### CODE REVIEW

File: docs/interfaces/mx4/accelerometer.md [hunk 1]
```diff
@@ -0,0 +1,103 @@
+---
+title: Accelerometer
+tags:
+    - C61
+    - MX-4
+    - MX-V
+    - MX-4 T30
+    - MX-4 T30fr
```
Comment (albsod) [inline docs/interfaces/mx4/accelerometer.md:8]:
```suggestion
    - T30
    - T30 FR
```

File: docs/interfaces/mx4/accelerometer.md [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+    - Linux 3.1
```
Comment (albsod) [inline docs/interfaces/mx4/accelerometer.md:9]: This suggests Linux 3.1-specific information, but then you also mention Linux 6.1.

File: docs/interfaces/mx4/accelerometer.md [hunk 3]
```diff
@@ -0,0 +1,103 @@
...
+This product is using MMA8452Q sensor and more information can be found at
+[NXP Semiconductors MMA8452Q Sensor](https://www.nxp.com/docs/en/data-sheet/MMA8452Q.pdf)
+
+
+## Wakeup
+Wake on Accelerometer - Not yet supported in software
+
+## For Linux 3.1 system
+
+### Wakeup
```
Comment (albsod) [inline docs/interfaces/mx4/accelerometer.md:23]: Unclear heading hierarchy.

File: docs/interfaces/mx4/accelerometer.md (hunk 2 above)
Comment (RikardO-HM) [reply docs/interfaces/mx4/accelerometer.md:9]: i was thinking about the chapter ## For Linux 3.1 system. for linux 4 and up iio is used

Comment (albsod) [reply docs/interfaces/mx4/accelerometer.md:9]: Ok, but my idea with tags is that they should be valid for all of the tags listed. If usage is different for some reason, then create a separate page where that specific information can be found. See e.g. mx4/leds

Comment (RikardO-HM) [reply docs/interfaces/mx4/accelerometer.md]: I removed the Linux 3.1

**Review by albsod — APPROVED** [review]

**Pull request merged**